.travis.yml

```diff
@@ -1,18 +1,28 @@
 language: ruby
 rvm:
-  - ruby-2.2
-
+- ruby-2.2
```

Comment:
When I moved VEC to travis last week, @plusjeff said "Instead of adding this, let's add a .ruby-version file. Travis will look at that, but it'll also make it easier to be in the correct Ruby version on the development side."  Not sure if we want to heed that advice here as well.